README.md

```diff
@@ -1,11 +1,24 @@
-# Speech Command Recognition
+# Honk: Speech Command Recognition
+## Speech commands demo
+Currently, PyTorch has official support for only Linux and OS X. Thus, Windows users will not be able to run this demo easily.
+
+To deploy the demo, run the following commands:
+- If you do not have PyTorch, please see [the website](http://pytorch.org).
+- Install Python dependencies: `pip install -r requirements.txt`
+- Install GLUT through your package manager (e.g. `apt-get install freeglut3-dev`)
```

Comment:
How hard would it be to pack all these things in a docker image, so I can just run the docker image in a single command without any setup required?

Reply:
I'll give it a shot.